tools/BUILD.bazel

```diff
@@ -0,0 +1,5 @@
+sh_binary(
```

Comment:
nit: consider adding a docstring on how this can be run locally
`bazel run --config asan --config clang tools:asan`?